style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -299,6 +304,12 @@ const metadata: StyleDictionary.MetadataIndex = {
     public: true,
     themeable: true,
   },
+  colorBorderSelectableItemFocused: {
```

Comment:
Would `colorBorderDropdownItemFocused` be a better name? We already have 6 public design tokens using the term `DropdownItem`, like `colorBorderDropdownItemDefault` or `colorBorderDropdownItemHover`. If this refers to the same kind of item, I think it should be named as `DropdownItem` too.